origin.spec

```diff
@@ -101,7 +101,13 @@ Requires:       docker-io >= %{docker_version}
 Requires:       tuned-profiles-%{name}-node = %{version}-%{release}
 Requires:       util-linux
 Requires:       socat
+# storage plugins
+Requires:       e2fsprogs
+Requires:       xfsprogs
 Requires:       nfs-utils
+Requires:       ceph-common
+Requires:       glusterfs-fuse
+Requires:       iscsi-initiator-utils
```

Comment:
We need both in origin because we're just ADDing the `openshift` binary into `openshift/origin` rather than installing it via RPM.  FWIW, these packages were always installed on RHEL hosts utilizing openshift-ansible.